openedx/features/content_type_gating/templates/content_type_gating/access_denied_message.html

```diff
@@ -12,7 +12,12 @@ <h3>
         {% if not mobile_app and ecommerce_checkout_link %}
         <span class="certDIV_1" style="">
             <a href="{{ecommerce_checkout_link}}" class="certA_1">
-                {% trans "Upgrade to unlock" %}  (${{min_price}} USD)
+                {% trans "Upgrade to unlock" %}
+                {% if price_before_discount %}
+                    (${{min_price}} <del>${{price_before_discount}}</del> USD)
```

Comment:
On the track selection code that I merged I had two prices including the currency where the entire currency string was crossed out  
like $10 USD ~~$20 USD~~  
I think this is what we tested though not sure.


We should probably have the same formatting though. Probably that would be a product decision?  
I would note that having the price with currency crossed out simplified things a lot for local currency. This way I just have each full price string in a span and the local currency code updates each span independently
If the currency was next to it, it would need special behavior to localize discounted prices and currency and it would need to know what that formatting looks like

Reply:
Ah, good call, I'll adjust that.